src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-office365.py

```diff
@@ -0,0 +1,19 @@
+# ------------------------------------------------------------------
+# Copyright (c) 2020 PyInstaller Development Team.
+#
+# This file is distributed under the terms of the Apache License 2.0
+#
+# The full license is available in LICENSE.APL.txt, distributed with
+# this software.
+#
+# SPDX-License-Identifier: Apache-2.0
+# ------------------------------------------------------------------
+
+"""
+Office365-REST-Python-Client contains xml templates that are needed by some methods
+This hook ensures that all of the data used by the package is bundled
+"""
+
+from PyInstaller.utils.hooks import collect_all
+
+datas, binaries, hiddenimports = collect_all("office365")
```

Comment:
`collect_all()` is extremely wasteful and should really only be used as a last resort. You only mention that it contains xml files so why not just use `collect_data_files()`?

Reply:
That works perfectly, thank you!